tests/js-tests/src/EffectsTest/EffectsTest.js

```diff
@@ -56,13 +57,27 @@ var EffecstsBaseLayer = BaseTestLayer.extend({
        this._super();
 
         var node = new cc.Node();
-	    var nodeGrid = new cc.NodeGrid();
+        node.setPosition(-winSize.width/2,0);
+        //if true
+        var visiablesize = director.getVisibleSize();
+        var gridRect = new cc.rect(visiablesize.width*0.1,
+                                   visiablesize.height*0.1,
+                                   visiablesize.width*0.5,
+                                   visiablesize.height*0.5);
+        //var actionGrid = new cc.GridAction();
+
+        var nodeGrid = cc.NodeGrid.create(gridRect); 
+
+        //if false
+        //var nodeGrid = new cc.NodeGrid();
+        cc.GridAction.prototype._gridNodeTarget = nodeGrid;
```

Comment:
You declared `_gridNodeTarget ` in EffectsBaseLayer, not in cc.GridAction.prototype